numpy/f2py/capi_maps.py

```diff
@@ -94,9 +96,9 @@
                   'complex_float': 'NPY_CFLOAT',
                   'complex_double': 'NPY_CDOUBLE',
                   'complex_long_double': 'NPY_CDOUBLE',
-                  'string':'NPY_STRING'
+                  'string': 'NPY_STRING',
+                  'character': 'NPY_STRING'}
```

Comment:
Should this perhaps be `NPY_CHAR` instead of `NPY_STRING` for `character`?

Reply:
No, the usage of `NPY_CHAR` is deprecated in numpy.